src/runtime/bytecode/astcode/register_location.zig

```diff
@@ -27,11 +27,11 @@ pub const RegisterLocation = enum(u32) {
     }
 
     pub fn fromIndexAllowNil(index: u32) RegisterLocation {
-        return @intToEnum(RegisterLocation, index);
+        return @as(RegisterLocation, @enumFromInt(index));
```

Comment:
Does this fail to infer the type?

Reply:
Good catch, I only fixed the casts. I'm gonna fix those cases as well.